src/main/webapp/app/exercises/programming/manage/update/programming-exercise-plans-and-repositories-preview.component.html

```diff
@@ -8,21 +8,15 @@
                 <hr class="pt-0 pb-0 mt-1 mb-1" />
                 <ul>
                     <li>
-                        <code class="text-white badge bg-primary label-preview">
-                            {{ getCourseShortName()?.toLowerCase() }}{{ programmingExercise.shortName.toLowerCase() }}-exercise
-                        </code>
+                        <code class="text-white badge bg-primary label-preview"> {{ shortName?.toLowerCase() }}{{ programmingExercise.shortName.toLowerCase() }}-exercise </code>
```

Comment:
Using the variable `shortName` instead of the method `getCourseShortName` as the method will be executed on each change (bad performance!)